Cargo.toml

```diff
@@ -124,46 +124,46 @@ unicode-normalization = { version = "0.1.22", default-features = false }
+sc-network-common = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-network-sync = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-offchain = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-rpc = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-service = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-sysinfo = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-telemetry = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-tracing = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-transaction-pool = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sc-transaction-pool-api = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sp-block-builder = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-blockchain = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-consensus = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sp-consensus-aura = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-genesis-builder = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-inherents = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-rpc = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-timestamp = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sp-transaction-pool = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
+sp-wasm-interface = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+substrate-build-script-utils = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+substrate-test-utils = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
 
-substrate-frame-rpc-system = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2412" }
-substrate-prometheus-endpoint = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2412" }
-sp-debug-derive = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2412", default-features = false }
+substrate-frame-rpc-system = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+substrate-prometheus-endpoint = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503" }
+sp-debug-derive = { git = "https://github.com/paritytech/polkadot-sdk", branch = "stable2503", default-features = false }
 
```

Comment:
Tagging this comment in here while I am thinking of it. It has been a while since we have updated either of these files that are just copied and slight modifications out of the template/polkadot-sdk. They might need to be updated? https://github.com/frequency-chain/frequency/tree/main/.maintain

Reply:
Was able to update one of them, and added a link to where we copied it from.
The other one, I can't seem to find the corresponding file in the polkadot-sdk repo; none of the likely candidates are a good enough match. Doesn't seem to hurt keeping the one we have for now...